packages/loot-core/webpack/webpack.browser.config.js

```diff
@@ -28,11 +28,6 @@ module.exports = {
       path: 'path-browserify',
     },
   },
-  resolveLoader: {
-    alias: {
-      'pegjs-loader': require.resolve('pegjs-loader'),
```

Comment:
Can we now do `yarn remove pegjs-loader`? :)

Reply:
Ignore this.. I'm used to a single `package.json` in the root directory. So when I did not find it there I got confused.

But you already have done this. Thanks!

ps. - leaving this for Jed to review. Pegjs is used for goal templates and I'm not a user of this feature.